superbench/benchmarks/model_benchmarks/model_base.py

```diff
@@ -12,7 +12,6 @@
 from superbench.benchmarks import Precision, ModelAction, DistributedImpl, DistributedBackend, BenchmarkType, ReturnCode
 from superbench.benchmarks.base import Benchmark
 from superbench.benchmarks.context import Enum
-from superbench.benchmarks.reducer import ReduceType
```

Comment:
do we still need the reducer after this PR?

Reply:
Reducer can be used for benchmarks which can't be modified to let only rank0 to generate the output.